infrastructure/main.tf

```diff
@@ -92,14 +92,16 @@ module "bulk-scan" {
     S2S_NAME   = "${var.s2s_name}"
     S2S_SECRET = "${data.vault_generic_secret.s2s_secret.data["value"]}"
 
-    REUPLOAD_BATCH             = "${var.reupload_batch}"
-    REUPLOAD_DELAY             = "${var.reupload_delay}"
-    REUPLOAD_MAX_TRIES         = "${var.reupload_max_tries}"
-    REUPLOAD_ENABLED           = "${var.reupload_enabled}"
-    SCAN_DELAY                 = "${var.scan_delay}"
-    SCAN_ENABLED               = "${var.scan_enabled}"
-    STORAGE_BLOB_LEASE_TIMEOUT = "${var.blob_lease_timeout}" // In seconds
-
+    REUPLOAD_BATCH                = "${var.reupload_batch}"
+    REUPLOAD_DELAY                = "${var.reupload_delay}"
+    REUPLOAD_MAX_TRIES            = "${var.reupload_max_tries}"
+    REUPLOAD_ENABLED              = "${var.reupload_enabled}"
+    SCAN_DELAY                    = "${var.scan_delay}"
+    SCAN_ENABLED                  = "${var.scan_enabled}"
+    STORAGE_BLOB_LEASE_TIMEOUT    = "${var.blob_lease_timeout}" // In seconds
+
+    QUEUE_ENVELOPE_SEND           = "${data.terraform_remote_state.shared_infra.queue_primary_listen_connection_string}"
```

Comment:
is this the correct variable from remote state?? (queue_primary_**listen**_connection_string)

Reply:
also, a suffix like` _CONN_STRING` on env var may clarify things a bit.